assets/texts/fr/amoDescription.html

```diff
@@ -11,6 +11,9 @@
 Il est basé sur le formidable projet <a href="https://missive.github.io/emoji-mart/">emoji-mart</a>. Vous pouvez déjà tester de nombreuses fonctionnalités sur le site, sans installer l'extension.
 Elle ne fonctionne que sur Firefox v.63 ou plus récent.
 
+Il a également une recherche intégrée, donc vous pouvez l'utiliser comme une "fiche de révision" d'émojis. <b>Trouvez
```

Comment:
Strange line wrapping (again?)…

Reply:
Argh, my bad.
I forgot to disable the code reformat at commit.